homeassistant/helpers/script.py

```diff
@@ -896,6 +906,27 @@ async def _async_error_step(self):
         trace_set_result(error=error)
         raise _AbortScript(error)
 
+    @async_trace_path("parallel")
+    async def _async_parallel_step(self) -> None:
+        """Run a sequence in parallel."""
+        # pylint: disable=protected-access
+        scripts = await self._script._async_get_parallel_scripts(self._step)
+
+        async def async_run_with_trace(idx: int, script: Script) -> None:
+            """Run a script with a trace path."""
+            path_stack_reset = trace_path_stack_cv.set(copy(trace_path_stack_cv.get()))
+            with trace_path([str(idx), "sequence"]):
+                await self._async_run_script(script)
+            trace_path_stack_cv.reset(path_stack_reset)
```

Comment:
Do you need to reset it? Since you set it inside this function invocation and then leave this function, I thought all changes would be reset anyway?

Reply:
No that is not needed, a quick test confirms it.
A bit of a leftover from an earlier approach, thank for spotting 👍